tasks/pgbouncer.yml

```diff
@@ -1,15 +1,18 @@
 ---
-- name: Add PgBouncer repository apt-key | apt
-  apt_key:
-    id: "{{ pgbouncer_apt_key_id }}"
-    url: "{{ pgbouncer_apt_key_url }}"
-    state: present
-  when: ansible_os_family == "Debian"
-
-- name: Add PgBouncer repository | apt
-  apt_repository:
-    repo: "{{ pgbouncer_apt_repository }}"
-    state: present
+# This will need replaced with deb822_repository eventually,
+# however module Requires ansible 2.15, ansible pacakage version for Centos7 is 2.9
+- name: Add PgBouncer apt repository | apt
```

Comment:
This block is for Debian systems only, but the comment mentions "Centos 7"

Reply:
The goal of the comment was that explain why the module `deb822_repository` was not used in favor of `get_url`. I probably need a little more context in the comment.

If the module `deb822_repository` was included in the role, any ansible controller with version less than 2.15 would throw an error about resolveing a module.

```
ERROR! couldn't resolve module/action 'deb822_repository'. This often indicates a misspelling, missing collection, or incorrect module path.

...

```


A Centos 7 instance being used as an ansible controller where ansible was installed from package would have version 2.9. For instance, an ansible run from an on premise centos7 all-in-one instance would fail, even though that module is not being used.

Reply:
Understood. 

Maybe "include_tasks" with condition on ansible version an/or OS distro will help. 

According to docs, the tasks are included dynamically during playbook execution (as opposed to import_tasks).

Reply:
That's a good idea!  I will test that and see if it has the desired result